### ISSUE TITLE
Operation Not Permitted On Android 12

### ISSUE THREAD

**Issue by Ep0chalypse** [laurent22/joplin#5691]
When I launch Joplin and open the sidebar and hit Synchronize it shows a error:

Completed: 11/07/2021 8:00 PM (0s)
Last error: Error: Error: Operation not permitted. Path: /storage/emulated/0/Notes/locks/exclusive_mobile_XXX.json

If I navigate to /storage/emulated/0/Notes/locks/ the folder is empty.

I have double checked Joplin's permissions and everything is granted(Camera, Files and media, Location).

## Environment

Joplin version: 2.6.1 
Platform: Android
OS specifics: Android 12 on Google Pixel 5a

**Comment by Robin-Sch** [comment]
same for android 11. Already tried reinstalling and deleting the folder

**Comment by betsythefc** [comment]
I've reinstalled Joplin on Android as well after encountering this issue and have the same problem.

**Comment by toddzilla57** [comment]
I'm also having this issue: Android 12, Pixel 4a. I tried downgrading Joplin, but it says the sync target is a newer version. I cannot use sync on Joplin Android at all currently.

**Comment by enboig** [comment]
The problem continues on joplin 2.6.9 and android 11
```

01-12T22:12:03,20,"Synchronizer: ""Error: Error: Operation not permitted. Path: /storage/emulated/0/Joplin/locks/2_2_775f718bf7b9490a8f15c910510ec709.json
Code: EUNSPECIFIED
value@index.android.bundle:1047:275
index.android.bundle:1047:4056
p@index.android.bundle:83:588
index.android.bundle:83:1893
p@index.android.bundle:83:588
o@index.android.bundle:83:1069
index.android.bundle:83:1241
u@index.android.bundle:77:157
index.android.bundle:77:866
index.android.bundle:85:1672
k@index.android.bundle:85:500
w@index.android.bundle:85:898
callReactNativeMicrotasks@index.android.bundle:85:3054
value@index.android.bundle:38:2868
index.android.bundle:38:960
value@index.android.bundle:38:2504
value@index.android.bundle:38:919
value@[native code]
value@[native code]""
```

**Comment by roman-r-m** [comment]
Same as https://github.com/laurent22/joplin/issues/5779

**Comment by D1Ck3n** [comment]
Got the same issue with my new Android 11 Device :-1:

**Comment by roman-r-m** [comment]
Check https://github.com/laurent22/joplin/issues/5779 there are some workarounds there

**Comment by PolGZ** [comment]
I have the same issue

> Check #5779 there are some workarounds there

As pointed here, I tried the "solution" of downgrading to v. 2.5.5 apk (https://github.com/laurent22/joplin-android/releases/download/android-v2.5.5/joplin-v2.5.5.apk), and it "worked".

It seems to be due to changes on permissions on newer versions of Android. Until somebody adapts it, we seem to need an older Joplin version. If you use Play Store, remember to dissable automatic updates until it is fixed.

**Comment by shumafuk** [comment]
Please, fix this. Latest update of Joplin on f-droid is 7 month old. I don't want to look for alternatives, but this issue makes Joplin sync with syncthing on Android completely unusable.

**Comment by tomasz1986** [comment]
> Joplin on f-droid

You really shouldn't be using this one. Joplin has been removed from F-Droid, so you're not going to see any updates there, at least for the time being. If possible, you should reinstall either from the Play Store or simply by using the official APK file.

**Comment by duleslo** [comment]
How to sync joplin **Android and PC**
1. Downgrade to version **joplin-v2.5.5.apk** (written in the post above, link)

![image](https://user-images.githubusercontent.com/56708444/212352764-d154cd1e-befc-4bd3-99e3-69e22b86d72e.png)

(Grant permissions to the directory on Android file system)
Set filesystem synchronization in joplin, and SAVE IT.
Install Syncthing on both devices PC and Android and set to syncthing
3. Install Joplin on Windows or Linux.
    (Set filesystem synchronization in joplin, and SAVE IT.)
4. Install Syncthing on both devices PC and Android and set to syncthing, 
   the same filesystem directory in which joplin should search for files
And that's how the thing works beautifully.

**Comment by treylav** [comment]
Unfortunately, the issue is still present in the version, available in the Play Store — 2.8.1 (2097668).

Despite the fact that a dialog box with request for permissions appears. in fact, Joplin does not request them or does it incorrectly: for example, it does not appear in the "Files" section of the Permission Manager.

At the same time, the version 2.5.5 suggested by @duleslo works, even with Storage Scopes.

If you need any help with tests on a client device, then I'm ready to provide it.

Specs:
 - Google Pixel 6A;
 - GrapheneOS (build TQ1A.230105.001.A2.2023012500) based on Android 13.
